FESS-GUI/mainwindow.cpp

```diff
@@ -106,9 +106,13 @@ MainWindow::MainWindow(QWidget *parent) :
     // setup a timer that repeatedly calls MainWindow::realtimeDataSlot:
     dataTimer = new QTimer(this);
     connect(dataTimer, SIGNAL(timeout()), this, SLOT(realtimeDataSlot()));
-    dataTimer->start(0); // Interval 0 means to refresh as fast as possible
-}
+    dataTimer->start(10); // Refresh every 10 milliseconds
```

Comment:
We should store the refresh rate in a variable, then reference that here and in velocitySlope()